pkg/apiserver/logsearch/models.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logsearch
+
+import (
+	"database/sql/driver"
```

Comment:
maybe not useful anymore?

Reply:
No it's used by [a function](https://github.com/pingcap-incubator/tidb-dashboard/pull/3/files#diff-9d74faa65ca1eeddd99f6ff22525a913R40).